### PR TITLE
NEW : collapse batch detail

### PR DESCRIPTION
**Pull request by atm-gauthier**

### CODE REVIEW

**Comment by atm-gauthier** [comment]
@eldy travis failed but error doesn't deal with what i added

File: htdocs/fichinter/list.php [hunk 1]
```diff
@@ -288,7 +288,7 @@
 	$sql .= " AND s.rowid = sc.fk_soc AND sc.fk_user = ".$user->id;
 }
 if ($socid) {
-	$sql .= " AND s.rowid = ".((int) $socid);
+	$sql .= " AND s.rowid = ".$socid;
```
Comment (eldy) [inline htdocs/fichinter/list.php:291]: Why removing the cast (it is a protection we should have when forging a sql with a test = on a non string data.

Comment (atm-gauthier) [reply htdocs/fichinter/list.php:291]: Hi @eldy
 it's a mistake, I will restore the cast on tomorrow ;)

Comment (atm-gauthier) [reply htdocs/fichinter/list.php]: @eldy it's ok now

**Comment by atm-gauthier** [comment]
@eldy everything's ok for you in this PR ?

**Pull request merged**